Signal/src/ViewControllers/InviteFlow.swift

```diff
@@ -146,7 +151,8 @@ class InviteFlow: NSObject, MFMessageComposeViewControllerDelegate, MFMailCompos
         return UIAlertAction(title: messageTitle, style: .default) { _ in
             Logger.debug("\(self.TAG) Chose message.")
             self.channel = .message
-            let picker = ContactsPicker(delegate: self, multiSelection: true, subtitleCellType: .phoneNumber)
+            let picker = ContactsPicker(allowsMultipleSelection: true, subtitleCellType: .phoneNumber)
+            picker.contactsPickerDelegate = self
```

Comment:
Passing a delegate in an initializer can be kind of inflexible/dodgy. 

In this particular case, I wanted the subclass to be it's own delegate, which isn't really possible without implicitly unwrapped optionals (which sometimes crash).

But also, there can be times when the delegate is unintentionally messaged before `self` has been completely initialized.

e.g. see the unfortunate `shouldNotifyDelegateOfUpdatedContacts` dance we do in ContactsViewHelper.

In general I'm trying to default towards always assigning the delegate after the fact.